lib/wanda/messaging/adapters/amqp/processor.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Wanda.Messaging.Adapters.AMQP.Processor do
+  @moduledoc """
+  AMQP processor.
+  """
+
+  alias Trento.Contracts
+  alias Wanda.Policy
+
+  @behaviour GenRMQ.Processor
```

Comment:
I would put this `behaviour` in top of all `alias`

Reply:
https://github.com/christopheradams/elixir_style_guide#module-attribute-ordering

Reply:
Changed as per style guide. Is it possible to configure the linter to check for this? I will look into it.

Reply:
@jamie-suse it is possible, but doing that has some downside. Sometimes there are macros that need to have a different order, so in my experience is a bit hard to enforce.
You could try to activate the credo check  (it's in the "controversial" section) and see what happens.